Documentation/contributing/development/contributing_guide.rst

```diff
@@ -235,43 +320,73 @@ Pull request review process
    | ``upgrade-impact``       | The code changes have a potential upgrade impact                          |
    +--------------------------+---------------------------------------------------------------------------+
 
-#. When everything looks OK, approve the changes.
+#. When all review objectives for all ``CODEOWNERS`` are met, all CI tests have
+   passed, and all reviewers have approved the requested changes, merge the PR
+   by clicking in the "Rebase and merge" button.
 
-#. When all review objectives for all ``CODEOWNERS`` are met and all CI tests
-   have passed, you may set the ``ready-to-merge`` label to indicate that all
-   criteria have been met.
+#. Merge PRs with the ``ready-to-merge`` label set `here <https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+sort%3Aupdated-asc+label%3Aready-to-merge+>`_
 
-   +--------------------------+---------------------------------------------------------------------------+
-   | Labels                   | When to set                                                               |
-   +==========================+===========================================================================+
-   | ``ready-to-merge``       | PR is ready to be merged                                                  |
-   +--------------------------+---------------------------------------------------------------------------+
+Triage issues for Triage team
+-----------------------------
+
+Dedicated expectation time for each member of Triage team: 15 / 30 minutes per
+day. Works best if done first thing in the working day.
+
+#. Committers belonging to the `Triage team <https://github.com/orgs/cilium/teams/triage>`_
+   should make sure that:
+
+   #. `Issues opened by community users are tracked down <https://github.com/cilium/cilium/issues?q=is%3Aissue+is%3Aopen+no%3Aassignee+sort%3Aupdated-desc>`_:
+
+       #. Add the label ``kind/community-report``;
+       #. If feasible, try to reproduce the issue described;
+       #. Assign a member that is responsible for that code section to that GitHub
+          issue;
+       #. If it is a relevant bug to the rest of the committers, bring the issue
+          up in the weekly meeting.
+
+   #. `Issues recently commented are not left out unanswered <https://github.com/cilium/cilium/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Akind%2Fcommunity-report>`_:
+
+       #. If there is someone already assigned to that GitHub issue and that
+          committer hasn't provided an answer to that user for a while, ping
+          that committer directly on Slack;
+       #. If the issue cannot be solved, bring the issue up in the weekly
+          meeting.
+
+Backporting PR for Backport team
+--------------------------------
+
+Dedicated expectation time for each member of Backport team: 60 minutes per
+week depending on release needs to be performed at the moment.
```

Comment:
```suggestion
week depending on releases that need to be performed at the moment.
```

Reply:
I think it's worth noting here that even if the next release is not imminently planned, it is still important to perform backports to keep the process smooth; If we delay backports, we may delay releases which we do not want.

In addition, stating that when you file a backport PR, you are responsible to drive it to completion, even if it stretches after your assigned week for backporters. If this is not feasible (eg PTO), you are responsible to initiate handover of the PR to the next week's backporters.